Source/EasyNetQ.Management.Client/Model/MessageStats.cs

```diff
@@ -5,10 +5,16 @@ namespace EasyNetQ.Management.Client.Model
     public class MessageStats
     {
         public Int64 DeliverGet { get; set; }
-        public DeliverGetDetails DeliverGetDetails { get; set; }
+        public MessageRateDetails DeliverGetDetails { get; set; }
         public Int64 DeliverNoAck { get; set; }
-        public DeliverNoAckDetails DeliverNoAckDetails { get; set; }
+        public MessageRateDetails DeliverNoAckDetails { get; set; }
         public Int64 Publish { get; set; }
-        public PublishDetails PublishDetails { get; set; }
+        public MessageRateDetails PublishDetails { get; set; }
```

Comment:
Added a couple of more Stats fields that I found in the HTTP API. But I couldn't find a comprehensive documentation so not sure if this is all of them and if there is a better way of structuring this..